### PR TITLE
added start and finishing block to queryFilter

### PR DESCRIPTION
**Pull request by personnamedmike**
To double check contract creation transactions, here's a list of block explorers:

- [ ] https://etherscan.io/
- [ ] https://goerli.etherscan.io/
- [ ] https://explorer.rsk.co/
- [ ] https://explorer.testnet.rsk.co/
- [ ] https://bscscan.com/
- [ ] https://testnet.bscscan.com/
- [ ] https://polygonscan.com/
- [ ] https://arbiscan.io/
- [ ] https://testnet.snowtrace.io/
- [ ] https://snowtrace.io/
- [ ] https://goerli.lineascan.build/
- [ ] https://mumbai.polygonscan.com/
- [ ] https://goerli.arbiscan.io/

### CODE REVIEW

File: src/contexts/lastLook/LastLook.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const LastLookContext = createContext<{
   subscribeAllServers(servers: Server[], pair: Pair): Promise<Pricing | any>[] {
     return [];
   },
-  unsubscribeAllServers: () => {},
+  unsubscribeAllServers: () => { },
```
Comment (personnamedmike) [inline src/contexts/lastLook/LastLook.tsx]: I don't know why this space is here. I don't see it locally

File: src/constants/firstTransactionBlocks.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+interface FirstTransactionBlocks {
+  [x: string]: { [x: number]: number };
+}
+
+export const firstTransactionBlocks: FirstTransactionBlocks = {
+  SwapERC20: {
+    1: 16776806, // Mainnet
```
Comment (piersss) [inline src/constants/firstTransactionBlocks.ts:7]:
```suggestion
import { ChainIds } from "@airswap/constants";

interface V3ContractsDeploymentBlock {
  SwapERC20: Record<ChainIds, number>;
  Wrapper: Record<ChainIds, number>;
  Registry: Record<ChainIds, number>;
}

export const firstTransactionBlocks: V3ContractsDeploymentBlock = {
  SwapERC20: {
    [ChainIds.MAINNET]: 16776806,
    // etc..
```

**Comment by piersss** [comment]
So we have to remember to maintain this every time to deploy? Well I guess there's no other way with current contract data from `@airswap/libraries`.

@dmosites Any way we can get the transaction hash of the contract deployment from `@airswap/libraries`?

**Comment by personnamedmike** [comment]
I don't think it's possible to use Ethers to directly get the deployment block. Would probably require a new constructor argument in the smart contract to make it possible

**Comment by dmosites** [comment]
Block number could be published with each package. Would you prefer:
1) a new file e.g. `@airswap/swap-erc20/deploys-blocks.js` or
2) via `deploys.js` used like `deploys[chainId].blockNumber` (and `deploys[chainId].address`)

**Comment by dmosites** [comment]
Ah I see what you mean by accessing these from libraries. Will take a look.

**Review by dmosites — APPROVED** [review]

**Pull request merged**